src/components/00-materials/README.md

```diff
@@ -12,17 +12,25 @@ npm install @axa-ch/materials
 
 ```js
 import { svg } from 'lit-html';
-import { ArrowRightSvg } from '@axa-ch/materials/icons';
+import { Arrow_forwardSvg } from '@axa-ch/materials/icons';
```

Comment:
Icons like `backspace.svg` gets exported as `BackspaceSvg`. 
We should consider exporting icons as `ArrowForwardSvg` (PascalCase) instead of `Arrow_forwardSvg`. This is just very inconsistent and not intuitiv.

Reply:
I agree.